### DISCUSSION TITLE
Flags documentation

### DISCUSSION THREAD

**Discussion by tatomir146** [Dynatrace/dynatrace-configuration-as-code#182] [Archive]
Discussion started in #162. As we are started implementing command based flags, and not to run into flags-hell, we should avoid using already established flags, if they are not serving same purpose, even if they are used in different command context. 

```
-v verbose
-e environments
-p projects
-c config or cluster
```

Would it be necessary to put this into a document? There is already inconsistency introduced where we are using kebab and camel case. The standard should be kebab. 

Please put your thoughts here so we can put this together as a document.

**Comment by CruzanCaramele** [comment]
I think with Radek's proposal [here](https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/discussions/186#discussioncomment-376252) we may want to consider the following:
 

- keep the following flags in both old and new CLI:
  -v  verbose
  -p  projects
  -d  dry-run
  
question comes to :

should we have **-c config** or **-c cluster**  . Personally it does not make too much meaning to me to have both **cluster** flag and **environment** flag when we completely move to the new CLI . Because **deploy** will just deploy to the **.yaml** file containing list of clusters or environments

In this case will it be better to have something like **-t target** to specify the file containing list of clusters or environments, ofcourse if this will not cause too much confusion

**Comment by ghost** [comment]
`-a` or `-api` for the `download` and `init` functionality. It would require rework but could this also be used for `deploy` so deployingparticular config types only rather than relying on `skipDeployment` ?
